R/caretEnsemble.R

```diff
@@ -127,13 +138,24 @@ caretEnsemble <- function(all.models, optFUN=NULL, ...){
 #' ens <- caretEnsemble(models)
 #' cor(predict(ens, newdata=iris[51:150,1:2]), iris[51:150,3])
 #' }
-predict.caretEnsemble <- function(object, keepNA = TRUE, se = FALSE, return_weights = FALSE, ...){
+predict.caretEnsemble <- function(object, keepNA = TRUE, level = 0.95, se = FALSE,
+                                  return_weights = FALSE, ...){
   stopifnot(is(object$models, "caretList"))
+  args <- eval(substitute(alist(...))) # get ellipsis values
+  args <- lapply(args, eval, parent.frame()) # convert from symbols to objects
+  if(exists("newdata", args)){
+    # tmp <- args$newdata
+    if(anyNA(args$newdata)){
```

Comment:
I'm pretty sure this line means that we will never get to the "Missing data found..." `keepNA` argument below, but I can't be sure because the data could be complete, but for some functions it could still produce an NA prediction. This seems pretty difficult to identify a test case for.